UndertaleModLib/UndertaleData.cs

```diff
@@ -78,7 +80,9 @@ public static UndertaleData CreateNew()
             data.FORM.Chunks["OBJT"] = new UndertaleChunkOBJT();
             data.FORM.Chunks["ROOM"] = new UndertaleChunkROOM();
             data.FORM.Chunks["DAFL"] = new UndertaleChunkDAFL();
+            data.FORM.Chunks["EMBI"] = new UndertaleChunkEMBI();
             data.FORM.Chunks["TPAG"] = new UndertaleChunkTPAG();
+            data.FORM.Chunks["TGIN"] = new UndertaleChunkTGIN(); // Maybe a studio 2 option may come in handy
```

Comment:
Yeah, CreateNew() right now generates a GMS1.4 file, so these shouldn't be here at all. But really, who would even attempt making a whole game from scratch in UndertaleModTool :P
```suggestion
```